packages/core/src/RenderingEngine/vtkClasses/vtkOffscreenMultiRenderWindow.js

```diff
@@ -39,6 +40,13 @@ function vtkOffscreenMultiRenderWindow(publicAPI, model) {
   model.openGLRenderWindow = vtkStreamingOpenGLRenderWindow.newInstance();
   model.renderWindow.addView(model.openGLRenderWindow);
 
+  // Create offscreen canvas and set it immediately
+  const offscreenCanvas = document.createElement('canvas');
+  offscreenCanvas.width = MAX_WIDTH;
```

Comment:
How many of these are we likely to have?  If tehre are a lot, it will consume a huge amount of memory.